label_sync/acm_repo/namespaces/batch_v1beta1_cronjob_label-sync-cron.yaml

```diff
@@ -0,0 +1,37 @@
+apiVersion: batch/v1beta1
```

Comment:
for my own education: what is the difference between this cron and cluster/label_sync_cron_job.yaml?

Reply:
The files in cluster is a kustomization package which is used to generate the "hydrated" manifests (kustomize build -o output).

We use kustomize because for the configmap we want to generate the configmap from an external file that contains the contents of the configmap but not the configmap itself.

Furthermore, we want the configmap to have a unique name based on the content hash of the configmap. This way any resource (e.g. a deployment) which references the configmap would change when the content of the config map changes which will cause new pods to be deployed. If we don't do this (i.e. if the configmap name is not unique based on the hash) then pods would not necessarily pick up the changed configmap.

By extension we use kustomize to generate the cron job because kustomize will correctly substitute in the generated name of the configmap which includes the unique content hash.